yarn.lock

```diff
@@ -990,6 +997,19 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@matterlabs/hardhat-zksync-solc@npm:^0.3.17":
+  version: 0.3.17
+  resolution: "@matterlabs/hardhat-zksync-solc@npm:0.3.17"
+  dependencies:
+    "@nomiclabs/hardhat-docker": ^2.0.0
```

Comment:
My only concern about this PR is how many libraries it adds to yarn.lock. It seems like hardhat-zksync-solc adds @nomiclabs/hardhat-docker as a dependency. I'm not sure why this requires this dependency...